features/other_b/series.feature

```diff
@@ -4,59 +4,53 @@ Feature: Create and Edit Series
   As a reader
   The index needs to load properly, even for authors with more than ArchiveConfig.ITEMS_PER_PAGE series
 
-  @disable_caching
-  Scenario: Three ways to add a work to a series
-    Given the following activated user exists
-      | login         | password   |
-      | author        | password   |
-      And a warning exists with name: "Choose Not To Use Archive Warnings", canonical: true
-      And the default ratings exist
-    When I am logged in as "author" with password "password"
-      And I go to the new work page
-      And I select "Not Rated" from "Rating"
-      And I check "Choose Not To Use Archive Warnings"
-      And I fill in "Fandoms" with "My Little Pony"
-      And I check "series-options-show"
-      And I fill in "work_series_attributes_title" with "Ponies"
-      And I fill in "Work Title" with "Sweetie Belle"
-      And I fill in "content" with "First little pony is all alone"
+  Scenario: Creator adds a work to a new series when the work is first posted
+    Given I am logged in as "author"
+      And I set up the draft "Sweetie Belle"
```

Comment:
I think it's hideous, honestly, especially since it uses the field id rather than the label (which I did because the label makes 0 sense out of context). :P 

Maybe the thing to do is change the Scenario description to say "Creator manually enters a series name to add a work to a new series when the work is first posted" and the other one can me "Creator selects an existing series name to add a work to an existing series when the work is first posted". That way, we can use the "as part of a series" bit.

Whatever @pixelred wants to do is fine by me!